app/api/metrics.py

```diff
@@ -1,11 +1,10 @@
-from app.schemas import ServiceMetricCreate
```

Comment:
if the two new imports here are useless, remove them please

Reply:
ok